app/views/devise/confirmations/show.html.slim

```diff
@@ -2,14 +2,14 @@
 
 
 h1.heading = t('forms.confirmation.show_hdr')
-p.italic Your password must be at least 8 characters.
+p#password_description.italic Your password must be at least 8 characters.
```

Comment:
i think it's best to avoid underscores in class and id names (https://developer.mozilla.org/en-US/docs/Web/Guide/CSS/Underscores_in_class_and_ID_Names)